server/config.js

```diff
@@ -129,6 +129,8 @@ module.exports = {
     },
     endpoints: {
       bookings: `${prisonApiBaseUrl}/api/bookings`,
+      offenders: `${prisonApiBaseUrl}/api/offenders`,
```

Comment:
Why have we chosen to abstract the API paths into config? I see we started in that direction before this PR, but it feels a bit odd to take on the extra complexity for something that isn't likely to change.

FWIW making the API base URL configurable makes perfect sense because this is likely to change across environments, this comment is just about the offenders endpoint.

Reply:
If I remember I _think_ this might be leftover from when we changed some of the APIs from the OG hub.

Reply:
This has been removed, and for the refactored code we have decided to avoid centralising API paths in the config in favour of picking up the environment variable directly in the constructor (see PrisonApiRepository)